pages/store/[id].vue

```diff
@@ -380,6 +385,47 @@ await callOnce(async () => {
 const bookInfo = useBookInfo({ nftClassId: nftClassId.value })
 const bookCoverSrc = computed(() => getResizedImageURL(bookInfo.coverSrc.value, { size: 600 }))
 
+const ogTitle = computed(() => {
+  const title = bookInfo.name.value
+  const author = bookInfo.authorName.value
+  return author ? `${title} - ${author}` : title
+})
+const ogDescription = computed(() => {
+  const description = bookInfo.description.value || ''
+  return description.length > 200 ? `${description.substring(0, 197)}...` : description
+})
+const canonicalUrl = computed(() => {
+  return `${baseURL}${route.path}`
+})
+
+const structuredData = computed(() => {
+  return generateBookStructuredData({
+    canonicalUrl: canonicalUrl.value,
+    selectedPricingItemIndex: selectedPricingItemIndex.value,
+    image: bookInfo.coverSrc.value,
```

Comment:
If image is from `useBookInfo`, this arg can be removed? since you already have used `useBookInfo` within the composable

Reply:
image is computed from `normalizeURIToHTTP` which calls `useRuntimeConfig()`, which somehow broke under this context
`normalizeURIToHTTP` might have to be a composable also, or somehow I have to prevent the lost of nuxt context, but I don't really know what is the issue with this context